tests/crowdsourcing/tasks/test_chat_demo.py

```diff
@@ -18,142 +18,73 @@
     "outputs": {
         "messages": [
             {
-                "packet_type": "update_status",
-                "sender_id": "mephisto",
-                "receiver_id": "1",
-                "data": {"state": {"agent_display_name": "Chat Agent 1"}},
-                "timestamp": 1604343628.610868,
+                "text": "Hi! How are you?",
```

Comment:
@meganung It looks like this change fixes TestChatDemo, so you may not need to take any further action on this PR